notifications/notification.py

```diff
@@ -1,27 +1,31 @@
-# -*- coding:utf-8 -*-
-import logging
-import smtplib
-import threading
+# -*- coding: utf-8 -*-
+
 import time
 from datetime import datetime, timedelta
+
+import jinja2
+import logging
+import smtplib
+from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 from email.utils import formatdate, make_msgid
-
 from sqlalchemy import text
 
+import config
 from database.models.recharge_event import RechargeEvent
 from database.storage import get_session
 from env import is_pi
 from users.users import Users
 
-SUBJECT_PREFIX = "[fd-noti] "
-MAIL_FROM = "flipdot-noti@vega.uberspace.de"
-
-# lower and we send mails every x days!
-minimum_balance = -5
-remind_mail_every_x_hours = 24 * 7
-
-FOOTER = """\n
+SUBJECT_PREFIX = "[fd-noti]"
+MINIMUM_BALANCE = -5
+REMIND_MAIL_EVERY_X_HOURS = 24 * 7
+FREQUENCIES = {
+    "instant": 0,
+    "daily": 60 * 60 * 24,
+    "weekly": 60 * 60 * 24 * 7,
+}
```

Comment:
Please re-add the new frequencies added recently: (lost while merging/rebasing?)

```
frequencies = {	
    "daily": 60 * 60 * 24,	
    "instant and daily": 60 * 60 * 24,	
    "weekly": 60 * 60 * 24 * 7,	
    "instant and weekly": 60 * 60 * 24 * 7,	
}
```

Also please check whether

Reply:
Keep `instant` only?

Reply:
Yes, we do.